docs/integrations/notification/_index.rst

```diff
@@ -42,9 +42,9 @@ The key can be found in the cluster configuration. For example it will be return
 Event Payload
 =============
 
-Currently we support two types of webhooks: ``Slack`` and ``Default``. A payload for a ``Default``
-webhook will contain information about the event itself, the trigger for the event, and the entity
-that triggered the event. The shape of ``event_data`` is determined by ``event_type``. Below is an
+For webhooks, you have two options: ``Slack`` and ``Default``. A payload for a ``Default`` webhook
```

Comment:
I don't think I like this better; it seems important to mention "type" as the answer to "options for _what_?". Perhaps "Determined supports two types of webhooks...".

Reply:
yes i like that

Reply:
edit applied